vision/object_detection_segmentation/ssd-mobilenetv1/README.md

```diff
@@ -0,0 +1,137 @@
+# SSD-MobilenetV1
+
+## Description
+
+SSD-MobilenetV1 is an object detection model that uses a Single Shot MultiBox Detector(SSD) approach to predict object classes for boundary boxes. 
+
+SSD enables the model to only need to take one single shot to detect multiple objects in an image, and mobilenet is a base network that provides high-level features for object detection. The combination of these two model frameworks produces an efficent, high-accuracy detection model that requires less computational powers.
+
+The SSD-MobilenetV1 is suitable for mobile and embedded based vision applications. 
+
+## Model
+
+|Model        |Download  | Download (with sample test data)|ONNX version|Opset version|
+|-------------|:--------------|:--------------|:--------------|:--------------|
+|SSD-MobilenetV1       | [29.3 MB](object_detection_segmentation/ssd-mobilenetv1/models/ssd_mobilenet_v1_coco_2018_01_28.onnx)  |[76.5 MB](object_detection_segmentation/ssd-mobilenetv1/models/ssd_mobilenet_v1_coco_2018_01_28.tar.gz) |1.7.0 | 10 |
```

Comment:
fix naming of the two files. Specifically, names like "ssd_mobilenet_v1_10.onnx" should be appropriate as we want to include the opset version in the file name.